engine/src/main/java/org/terasology/rendering/openvrprovider/OpenVRProvider.java

```diff
@@ -70,6 +70,19 @@
     private final VRTextureBounds_t texBounds = new VRTextureBounds_t();
     private float nearClip = 0.5f;
     private float farClip = 500.0f;
+    private static OpenVRProvider instance = null;
```

Comment:
CheckStyle should have caught this: on declaration this variable is null anyway. This line can just be:

`private static OpenVRProvider instance;`